docs/TuxAnimations.md

```diff
@@ -0,0 +1,237 @@
+# Tux/Animations
+
+##Premise
+>Tux Animations is a library of default animation components and easings. Additionally Tux Animations includes the createAnimationClass function which can be used to create custom animation components.
+
+##Implementation
+####1) Requiring Animations Components
+The Tux Animations interface including all animation defaults can be exposed via:
+
+```
+  var Animations = require('tux/Animations');
+```
+
+#####Require only what you need
+Tux is a modular framework and as such it is possible to require only the minimal number of files needed in order to save space in your app. For Example, if you need access to the default Fly and FadeUp animations in your implementation, you need only to require those specific animations via:
+
+```
+  var Fly = require('tux/Animations/Fly');
+  var FadeUp = require('tux/Animations/Fade/Up');
+```
+
+####2) Rendering Animation Components
+The only other step to implementing a Tux animation is to wrap the element you want animated with the desired animation component within the render method of your ``React.createClass`` function.
```

Comment:
You only need one set of \`'s for markdown.  So it should be \`React.createClass\`.